pkgs/tools/networking/http2tcp/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, python3Packages
+, fetchurl
+}:
+
+python3Packages.buildPythonApplication rec {
+
+  pname = "http2tcp";
+  version = "0.5";
```

i would move format up here
```suggestion
  version = "0.5";
  format = "other";
```